### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by Timowest12**
Added the contact form. Also added functionality trough Formspree.

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -297,3 +307,30 @@ nav > ul > li {
...
+  padding: 10px;
+}
+
+.btnwrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+footer {
```
Comment (elisha2kyakpo1) [inline styles/style.css:330]: - [ ] There is a page overflow in the X-axis, I don't know if this was there even in other milestones, but it has to be fixed.

overflow
overflow2

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi, @Timowest12,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- The form requirements are implemented properly.👏👏👏
- The branch name is descriptive.👏👏👏

## Required Changes ♻️
_Check the comments under the review._

- [ ] Provide descriptive(step by step information on how to clone or get a copy of the project and running it in a code editor, for example, git clone <REPO_NAME>, which button or URL to copy, how to navigate to the folder containing the project, etc)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Timowest12** [comment]
Thanks for the feedback. I don't know either where that overflow came from but i have been managed to solve it!

**Comment by Timowest12** [comment]
Thanks for the review, I do not know why I suddenly had that overflow either, but luckely i have been able to solve it.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi, @Timowest12 and @sja-thedude, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**